### PR TITLE
Add autodiscover example for Auditbeat

### PR DESCRIPTION
**Pull request by jsoriano**
Add autodiscover example to monitor the file integrity of files in containers.

Add `NODE_NAME` environment variable, that is needed for the example
and is commonly used in kubernetes config snippets.

### CODE REVIEW

File: deploy/kubernetes/auditbeat-kubernetes.yaml [hunk 1]
```diff
@@ -150,6 +172,10 @@ spec:
         hostPath:
           path: /var/lib/auditbeat-data
           type: DirectoryOrCreate
+      - name: run-containerd
+        hostPath:
+          path: /run/containerd
+          type: DirectoryOrCreate
```
Comment (exekias) [inline deploy/kubernetes/auditbeat-kubernetes.yaml:178]: what happens here if you are not using containerd?

Comment (jsoriano) [reply deploy/kubernetes/auditbeat-kubernetes.yaml:178]: An empty directory is created. I considered commenting out the code and explaining that it has to be commented in if file_integrity is used with the files of the containers, but I think this is easier this way for the user and not so bad in general.
When using docker as runtime this directory is also created, but it doesn't contain the root filesystems.
With cri-o this directory doesn't exist, so with this config it is created empty.

Comment (exekias) [reply deploy/kubernetes/auditbeat-kubernetes.yaml:178]: Works for me, I wonder if all this deserves some mention in https://www.elastic.co/guide/en/beats/auditbeat/master/running-on-kubernetes.html

Comment (jsoriano) [reply deploy/kubernetes/auditbeat-kubernetes.yaml:179]: Added a note about this in the docs.

**Comment by jsoriano** [comment]
@exekias can you take a quick look to the changes in docs? Thanks!

File: auditbeat/docs/running-on-kubernetes.asciidoc [hunk 2]
```diff
@@ -88,3 +88,11 @@ $ kubectl --namespace=kube-system get ds/{beatname_lc}
...
 {beatname_lc}   32        32        0         32           0           <none>          1m
 ------------------------------------------------
+
+[WARNING]
+=======================================
+{beatname_uc} is able to monitor the file integrity of files in pods, to be able
+to do that, the directories with the container root file systems have to be
+mounted as volumes in the {beatname_uc} container. For example, containers
+executed with containerd have their root file systems under `/run/containerd`.
+=======================================
```
Comment (exekias) [inline auditbeat/docs/running-on-kubernetes.asciidoc:98]: What do you think about putting an example here or referencing the manifest to see it?

Comment (jsoriano) [reply auditbeat/docs/running-on-kubernetes.asciidoc:99]: Added a mention and link to the reference manifest.

**Pull request merged**